tests/testthat/test_Select.R

```diff
@@ -0,0 +1,17 @@
+test_that("selectors work", {
+  n_epochs = 1
+
+  task = tsk("iris")
+
+  mlp = lrn("classif.mlp",
```

Comment:
no need to use parameter names for the test here. The functionalities are really orthogonal